style.css

```diff
@@ -0,0 +1,632 @@
+@import url("https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&family=Roboto:wght@300&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;500;600;700&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Lato", sans-serif;
+  color: #272a31;
+}
+
+.mobile-nav {
+  padding: 20px;
+  display: flex;
+  position: absolute;
+}
+
+.Welcome-container {
+  background-image: url(./images/phone-view-icons/mobile_intro_bg.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  padding: 60px 25px 25px 20px;
+}
+
+h1 {
+  color: #ec5242;
+  text-align: left;
+  margin-bottom: 10px;
+  font-family: "Montserrat", sans-serif;
+  font-size: 18px;
+  font-weight: 900;
+}
+
+h2 {
+  color: #ec5242;
+  background: url(./images/phone-view-icons/tools-for-motivation-KvTOwKoji7g-unsplash.jpg);
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  font-size: 40px;
+  font-family: "Lato", sans-serif;
+  font-weight: 900;
+}
+
+.intro-para {
+  color: #4e4e50;
+  border: 4px solid #ffff;
+  background-color: #f8f5f5;
+  width: 75%;
+  padding: 10px;
+}
+
+.second-para {
+  color: #272a31;
+  font-weight: bolder;
+  font-size: 30px;
+  margin-top: 14px;
+}
+
+.thierd-para {
+  color: #272a31;
+  font-family: "Lato", sans-serif;
+}
```

Comment:
-Kindly, fill out the headline content and follow some color and font-weight to stick the given design and have better UI 👍  

| Figma design                                                             | Your design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://github.com/axcell-0/first-capstone/assets/52010789/8e0ef062-143c-496b-9fd7-ab1858313ee8" width="200">|<img src="https://github.com/axcell-0/first-capstone/assets/52010789/7cdf3f15-58a2-4b04-b87b-e67fadfeb3bf" width="200">|


😎️

🤝️